include/networkit/graph/Graph.hpp

```diff
@@ -819,6 +819,12 @@ class Graph final {
      */
     node addNode();
 
+    /**
+     * Adds n nodes.
+     * @return The index of the last node added.
+     */
+    node addNodes(node numberOfNewNodes=1);
```

Comment:
Type of `numberOfNewNodes`: should we use `count`? Also, do not forget to document `numberOfNewNodes` (`@param numberOfNewNodes ...` is missing).

Reply:
I prefer the original name, as its clear without doubt that the number of new nodes (rather than the total number of nodes; does not seem plausible, but in doubt, one has to check) has to be provided. Backed up by [Kate G.'s talk on CppCon](https://www.youtube.com/watch?v=MBRoCdtZOYg) I'm currently leaning towards longer names ;)

Reply:
I think Eugenio was referring to the type here.

Reply:
Should we also have a `resizeNodes` function?

Reply:
That's indeed a good talk ;) however, my point was about the type, not the name. I am totally fine with the chosen name.

Reply:
That makes more sense. Thank's for clarifying. I changed it. Somewhat related: If we progress with 32bit ids (#427), we should add a check to `addNode` and `addNodes` not to overflow.

Regarding `resizeNodes()`: Shrinking is a little more involved as growing, since we have to remove edges to non-existings nodes. I have a PR that introduces an `removeEdgeIf` method that could be used, but are waiting until #421 due to some conflicts. Hence I suggest to post-pone adding `resizeNodes()`.

Reply:
Thank you. Yes if we make progress with 32 bit ids we should add those checks.